mne/io/edf/edf.py

```diff
@@ -506,6 +506,20 @@ def _get_info(fname, stim_channel, eog, misc, exclude, preload):
     return info, edf_info, orig_units
 
 
+def _parse_prefilter_string(prefiltering: str):
+    """Parses prefilter string from EDF+ and BDF headers for highpass
+       and lowpass filters"""
+    highpass = np.array(
+        [v for hp in [re.findall(r'HP:\s*([0-9]+[.]*[0-9]*)', filt)
+                      for filt in prefiltering] for v in hp]
+    )
+    lowpass = np.array(
+        [v for hp in [re.findall(r'LP:\s*([0-9]+[.]*[0-9]*)', filt)
+                      for filt in prefiltering] for v in hp]
+    )
+    return highpass, lowpass
```

Comment:
Now that it's a private function, you can directly test that old-style and new-style strings work, e.g. in a new `test_edf_prefilter_parse` function in `mne/io/edf/tests/test_edf.py`

Reply:
Gotcha, can do.